### DISCUSSION TITLE
`series_name` doesn't work as expected

### DISCUSSION THREAD

**Discussion by markus-hinsche** [Project-MONAI/MONAI#5571] [Q&A] [answered]
I have a directory full of DICOM files of different series: 
* series "38" has 96 slices, 
* series "34" has 96 slices, 
* series "46" has 88 slices, 
* series "42" has 88 slices,

I use the following code to read
```python
tfm = monai.transforms.LoadImage(series_name='42', series_meta=True)
pixel_arr, meta = tfm("path/to/dir/with/dicoms")
```

Since I am dealing with DICOM files(don't have .dcm ending though), I would expect  [ITKReader](https://docs.monai.io/en/stable/data.html#monai.data.ITKReader) to be the reader that will be used.

However I get the following output: https://gist.github.com/markus-hinsche/c78278f462b10d72b4ccdd9cd14be712

I think the reason is that `name = names_generator.GetFileNames(series_identifier)` in 
https://github.com/Project-MONAI/MONAI/blob/b030839e98e6a00c1ab0e53545f60b62dc4da4a4/monai/data/image_reader.py#L265
sets `name = ()`(empty tuple) because `GetFileNames` doesn't work as expected.

In contrast when running

```python
tfm = monai.transforms.LoadImage()
pixel_arr, meta = tfm("path/to/dir/with/dicoms")
```

I get 96 slices (because pixel_arr shape is `(320, 260, 96)`), so it reads the first series.

Any idea how I can get a specific series name?

**Comment by kbressem** [comment, marked as answer]
ITKReader accepts the argument `series_name`. 

You can initialize the reader and pass it to LoadImage. 

https://docs.monai.io/en/stable/data.html#itkreader
> series_name (str) – the name of the DICOM series if there are multiple ones. used when loading DICOM series.
